mist/runtime/loss.py

```diff
@@ -31,6 +31,64 @@ def forward(self, y_true, y_pred):
         loss = torch.mean(loss)
         return loss
 
+#Implement clDice:
+class soft_cldice(nn.Module):
+    def __init__(self, iter_=3, smooth = 1., exclude_background=False):
+        super(soft_cldice, self).__init__()
+        self.iter = iter_
+        self.smooth = smooth
+        self.soft_skeletonize = SoftSkeletonize(num_iter=10)
+        self.exclude_background = exclude_background
+
+    def forward(self, y_true, y_pred):
```

Comment:
For readability, please space this code out and add comments saying what is going on in this function.

Reply:
Same comment here:

Also, add these lines here:

```
# Prepare inputs
y_true = get_one_hot(y_true, y_pred.shape[1])
y_pred = softmax(y_pred, dim=1)
```

We apply `get_one_hot` because the data loader does not stream the masks in one hot encoder and the networks don't apply a final activation to their outputs. This latter part gives us more flexibility for future applications like regression instead of segmentation.